community/cypher/runtime-util/src/main/scala/org/neo4j/cypher/internal/runtime/planDescription/LogicalPlan2PlanDescription.scala

```diff
@@ -405,6 +405,13 @@ case class LogicalPlan2PlanDescription(readOnly: Boolean, cardinalities: Cardina
             val lessThanBoundsText = lessThanBounds.bounds.map(bound =>
               s"<${bound.inequalitySignSuffix} ${bound.endPoint.asCanonicalStringVal}").toIndexedSeq
             (name, InequalityIndex(label.name, propertyKey, greaterThanBoundsText ++ lessThanBoundsText))
+          case PointDistanceSeekRangeWrapper(PointDistanceRange(point, distance, inclusive)) =>
+            val poi = point match {
+              case FunctionInvocation(Namespace(List()), FunctionName("point"), _, Seq(MapExpression(args))) =>
```

Comment:
In a couple of places, we have used the hard coded strings for function names. I think it would be better if we instead used something like `org.neo4j.cypher.internal.v3_4.functions.Point.name`

Reply:
I have only found this one place...